### DISCUSSION TITLE
Is there a way to view custom fields?

### DISCUSSION THREAD

**Discussion by smason-apex** [ankitpokhrel/jira-cli#689] [Q&A]
We make heavy use of custom fields in our instance. It'd be great if they could be shown via `jira issue view`.

Is this possible to do with templates? Or some other way?

**Comment by ankitpokhrel** [comment]
Hi @smason-apex, this is not possible at the moment. I'll work on this in the near future, PRs are welcome in the mean time. Thanks!
